front/pages/api/w/[wId]/spaces/[spaceId]/trackers/[tId]/index.ts

```diff
@@ -166,5 +166,5 @@ async function handler(
 }
 
 export default withSessionAuthenticationForWorkspace(
-  withResourceFetchingFromRoute(handler, "space")
+  withResourceFetchingFromRoute(handler, { space: { requireCanList: true } })
```

Comment:
For non v1 endpoint we do have a few requireCanList but most of them should be requireCanRead no?

Reply:
that one should be canRead. I'll review all of them